website/siteConfig.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config.html for all the possible
+// site configuration options.
+
+/* List of projects/orgs using your project for the users page */
+const users = [
+  {
+    caption: 'GitLab',
+    image: 'https://gitlab.com/gitlab-com/gitlab-artwork/raw/master/logo/logo-square.png',
+    infoLink: 'https://gitlab.com/',
+    pinned: true,
+  },
+];
+
+const siteConfig = {
+  title: 'KaTeX',
+  tagline: 'The fastest math typesetting library for the web',
+  url: 'https://khan.github.io',
+  baseUrl: '/KaTeX/',
+
+  // Used for publishing and more
+  projectName: 'KaTeX',
+  organizationName: 'Khan',
+
+  headerLinks: [
+    {doc: 'node', label: 'Docs'},
+    {page: 'users', label: 'Users'},
+    {href: 'https://github.com/Khan/KaTeX', label: 'GitHub'},
+    {search: true},
+  ],
+
+  // If you have users set above, you add it here:
+  users,
```

Comment:
So this gets passed to the `Users` component defined in `users.js`?

Reply:
@kevinbarabash Yes, any property can be exported through `siteConfig` and used in other components.

Reply:
Cool!  It seems like docusaurus is quite flexible.